From 6051b954c0822f03169e67d8b47d741f44a4814a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Fri, 18 Nov 2016 22:00:23 +0800
Subject: [PATCH 1279/5242] MLK-13480-2 ARM: dts: imx7ulp: change TPM clock
 parent to OSC

commit  178273c17a10b90ba05ceb726c78ca5e76e77fee from
https://source.codeaurora.org/external/imx/linux-imx.git

As FIRC may be NOT accurate enough and it also can
be disabled when M4 goes into VLPR mode, so using
OSC as TMP clock parent is better.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 2bfbe61..056363c 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -174,8 +174,6 @@
 			clocks = <&clks IMX7ULP_CLK_LPTPM5>,
 				 <&clks IMX7ULP_CLK_NIC1_BUS_DIV>;
 			clock-names = "per", "ipg";
-			assigned-clocks = <&clks IMX7ULP_CLK_LPTPM5>;
-			assigned-clock-parents = <&clks IMX7ULP_CLK_FIRC>;
 		};
 
 		lpit: 1@40270000 {
@@ -328,6 +326,8 @@
 			clock-names = "ckil", "osc", "sirc",
 				"firc", "upll", "mpll";
 			#clock-cells = <1>;
+			assigned-clocks = <&clks IMX7ULP_CLK_LPTPM5>;
+			assigned-clock-parents = <&clks IMX7ULP_CLK_OSC>;
 		};
 
 		pcc2: pcc2@403F0000 {
-- 
1.7.9.5

